tests/test_imsimCmpt.py

```diff
@@ -248,7 +248,7 @@ def testMapOpdDataToListOfWfErr(self):
             self.assertEqual(sensorId, mapSensorNameAndId[refSensorName])
 
             zkInWfErr = wfErr.annularZernikePoly
-            delta = np.sum(np.abs(zkInWfErr - opdZk[sensorId]))
+            delta = np.sum(np.abs(zkInWfErr - opdZk[sensorId] / 1e3))
```

Comment:
Same as above - could you comment why is this unit change happening?

Reply:
Same as above. Will add same comment here.